app/javascript/app.spec.js

```diff
@@ -1,14 +1,14 @@
-import { shallowMount } from '@vue/test-utils';
-import App from 'app';
+import { shallowMount } from "@vue/test-utils";
+import App from "app";
 
-describe('App', () => {
-  test('is a Vue instance', () => {
+describe("App", () => {
+  test("is a Vue instance", () => {
     const wrapper = shallowMount(App);
     expect(wrapper.isVueInstance()).toBeTruthy();
   });
 
-  it('displays message on load', () => {
+  it("displays message on load", () => {
     const wrapper = shallowMount(App);
-    expect(wrapper.find('p').text()).toEqual('Hello Vue!');
+    expect(wrapper.find("p").text()).toEqual("Hello Vue!");
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'expect' is not defined       no-undef